doc/source/cluster/vms/references/ray-cluster-configuration.rst

```diff
@@ -958,6 +961,19 @@ If enabled, nodes will be *stopped* when the cluster scales down. If disabled, n
 * **Type:** Boolean
 * **Default:** ``True``
 
+.. _cluster-configuration-use-internal-ips:
+
+``provider.use_internal_ips``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If enabled, Ray will use private IP addresses for communication between nodes.
+This should be omitted if your network interfaces use public IP addresses.
+
+* **Required:** No
+* **Importance:** Low
+* **Type:** Boolean
+* **Default:** ``False``
```

Comment:
I had to look this up in the codebase where it was hardcoded in six or seven places.  If you like I can pull this out into a constant `AUTOSCALER_USER_INTERNAL_IPS_DEFAULT`.